functions/error-reporting.ts

```diff
@@ -24,6 +25,15 @@ export const handler: Handler = async ({ body }) => {
       user,
     } = JSON.parse(body)
     Bugsnag.notify({ name, message, stack, cause }, (event) => {
+      event.app = {
+        releaseStage: 'production',
+        version: cliVersion,
+        type: 'netlify-cli',
```

Comment:
type should be set to `node`

Reply:
Does not need to be you can give any name to it (default is `node`)

Reply:
but why `netlify-cli` what's the meaning of setting this?

Reply:
<img width="852" alt="CleanShot 2023-04-19 at 17 14 03@2x" src="https://user-images.githubusercontent.com/11156362/233120967-1d9702fc-f4b1-4d35-8dbf-a1f2df18cd66.png">

build-info can report to bugsnag as well, so we can differentiate from where this was done. 

So we can report to the same project from multiple sub parts:

that's what they outline here: 
<img width="500" alt="CleanShot 2023-04-19 at 17 15 01@2x" src="https://user-images.githubusercontent.com/11156362/233121320-5eefb40e-b653-4d6c-94bb-af7d27691b9a.png">